extension/options-new.js

```
@@ -451,7 +499,9 @@ document.addEventListener('DOMContentLoaded', async () => {
   
   // Test connection button
   elements.testButton.addEventListener('click', async () => {
-    const apiKey = elements.apiKey.value.trim();
+    const apiKey = provider === 'google'
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)